### DISCUSSION TITLE
Suggestion ClaimExplosions

### DISCUSSION THREAD

**Discussion by Jorrevdb** [GriefPrevention/GriefPrevention#1299] [Ideas]
As an Owner of a survival server
My players are requesting that only the players with Permissiontrust can Change the Claimexplosions
Because somethimes they /trust a player that they don't "Trust" 100%. And if that player want to grief that claim he can set Claim explosions on true so that he can blow up the base with tnt realy easy

Maybe you can add it to the config, so that other admins can choose if players need Permission trust or Normal trust to allow Tnt Explosions

**Comment by Jikoo** [comment]
Duplicate of #618. My views can be found [here](https://github.com/TechFortress/GriefPrevention/discussions/618#discussioncomment-194004).
